packages/blocks/src/api/types.ts

```diff
@@ -0,0 +1,55 @@
+/**
+ * Describes options for Block serialization.
+ *
+ * Used by the Block [serializer](./serializer.js).
+ *
+ * @public
+ */
+export type BlockSerializationOptions = {
+	/**
+	 * TODO Undocumented
+	 */
+	isCommentDelimited?: boolean;
+
+	/**
+	 * TODO Undocumented
```

Comment:
```suggestion
	 * Whether we are serializing inner blocks.
```

Reply:
@tyxla what do you think of this? Is this option only ever used inside the `serialize` function when descending into the inner blocks?

```suggestion
	 * Whether the serialization process has descended into the inner blocks.
```

Reply:
Feel free to go through your suggestion 👍 I think it's more descriptive than what I came up with. In any case, either is much better than having nothing after all.